client/src/pages/Therapist/ClientFeedback/index.js

```diff
@@ -0,0 +1,77 @@
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+
+import * as S from './style';
+import * as T from '../../../components/Typography';
+import { Row, Col } from '../../../components/Grid';
+import Button from '../../../components/Button';
+import SubHeader from '../../../components/SubHeaderWithGreenLine';
+
+import { TherapistClients } from '../../../api-calls';
+import { userStatuses } from '../../../constants';
+import Title from '../../../components/Title';
+const ClientFeedback = ({ clientHistory }) => {
```

Comment:
```suggestion
const ClientFeedback = () => {
```
unused var, though we should ask the client history from the back not passing it as a prop

Reply:
done